examples/tests/cloud-push-pull/azure-container-registry.earth

```diff
@@ -0,0 +1,12 @@
+FROM earthly/dind:alpine-main
+
+push:
+    RUN date | md5sum | head -c 32 > motd
```

Comment:
Should make a unique file; unless we run the same target twice in a second. Force different layers to _actually_ push.